backend/cmm.mli

```diff
@@ -165,7 +165,8 @@ type memory_chunk =
   | Thirtytwo_signed
   | Word_int                           (* integer or pointer outside heap *)
   | Word_val                           (* pointer inside heap or encoded int *)
-  | Single
+  | Storage_single                     (* F64 in registers, F32 in memory. *)
+  | Real_single                        (* F32 in both registers and memory. *)
```

Comment:
Why do we need a different `memory_chunk` if the memory representation is the same?

Reply:
These get propagated all the way to `emit.mlp`, which maps `Storage_single` (same as the previous `Single`) to `cvtss2sd` and `Real_single` to `movss`.